Riot/Categories/UIViewController.swift

```diff
@@ -27,10 +27,9 @@ extension UIViewController {
     /// Remove back bar button title when pushing a view controller.
     /// This method should be called on the previous controller in UINavigationController stack.
     @objc func vc_removeBackTitle() {
-        self.navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        self.navigationItem.backButtonDisplayMode = .minimal
```

Comment:
Haha! https://github.com/vector-im/element-x-ios/pull/86#discussion_r902674969

👏👏👏

Reply:
Yeah I suppose we couldn't just do this back in the day with iOS 13 support 😛

Reply:
Yeah we couldn't just do this back in the day with iOS 13 support.